cmd/milmove/gen_duty_stations_migration.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+
+	"github.com/transcom/mymove/pkg/cli"
+	"github.com/transcom/mymove/pkg/logging"
+	dutyStations "github.com/transcom/mymove/pkg/services/duty_stations"
+)
+
+const (
+	// filename containing the details for new office users
+	DutyStationsFilenameFlag string = "office-users-filename"
```

Comment:
Should the flag here be `duty-stations-filename`?

```suggestion
	DutyStationsFilenameFlag string = "duty-stations-filename"
```